cli/server_createadminuser.go

```diff
@@ -197,6 +197,7 @@ func (r *RootCmd) newCreateAdminUserCommand() *serpent.Command {
 					UpdatedAt:      dbtime.Now(),
 					RBACRoles:      []string{rbac.RoleOwner().String()},
 					LoginType:      database.LoginTypePassword,
+					Status:         "",
```

Comment:
why do we pass `""` here? Should we just pass `active`?

Reply:
Ideally I would omit it but the linter doesn't like leaving it out. It doesn't really matter what this is created as, so `""` just does the default behavior which is dormant.